src/DynamoCoreWpf/ViewModels/Menu/PreferencesViewModel.cs

```diff
@@ -1428,6 +1429,11 @@ public PreferencesViewModel(DynamoViewModel dynamoViewModel)
             InitializeCommands();
         }
 
+        private void Model_ShutdownStarted(DynamoModel model)
+        {
+            dynamoViewModel.MainGuideManager?.CloseRealTimeInfoWindow();
```

Comment:
why is this being called from the preferenceViewModel? IE why is the preferences view model in control of this. It seems pretty strange to me that PreferencesViewModel is gluing these together.

Reply:
Maybe it should be at where we close and dispose `dynamoViewModel`